starfish/test/full_pipelines/cli/_base_cli_test.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import unittest
+from typing import List, Tuple
+
+from starfish.intensity_table import IntensityTable
+from starfish.util import clock
+
+
+class CLITest(unittest.TestCase):
+    __test__ = False
+
+    SUBDIRS = Tuple[str]
```

Comment:
Check with @ttung about how to make it clear that these are abstract. 

Also this class needs docblocks explaining how it should be extended to create more tests. :-)

Reply:
Oh yeah.  You can make it a `@property` method that raises `NotImplementedError()`.

Subclasses will then return what's expected.